pusher-platform-core/src/main/kotlin/com/pusher/platform/subscription/BaseSubscription.kt

```diff
@@ -119,13 +108,13 @@ internal class BaseSubscription<A>(
         return (this as? Result.Success)?.value
     }
 
-    private val ResponseBody.messages
+    private val ResponseBody.messages: Sequence<Result<SubscriptionMessage<A>, Error>>
         get() = charStream().buffered().lineSequence()
-            .map {line -> line.toSubscriptionMessage<A>(messageParser) }
+            .map {line -> line.toSubscriptionMessage(messageParser) }
 
     override fun unsubscribe() {
         call.takeUnless { it.isCanceled }?.cancel()
+        job.takeUnless { it.isCancelled }?.cancel()
```

Comment:
How many `l`s does cancelled have? Cos these two lines disagree...

Reply:
Depends on which country you are from 😂  :
https://www.grammarly.com/blog/canceled-vs-cancelled/
The first one is from OkHttp: https://square.github.io/okhttp/3.x/okhttp/okhttp3/Call.html#isCanceled--
The second one if from Java: https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/Future.html#isCancelled()

Reply:
The convention is to go with the american spelling for consistency, but when two american organisations can't agree... :man_facepalming: